test/integration/error_spam_test.go

```diff
@@ -34,6 +34,8 @@ var stderrWhitelist = []string{
 	`slow|long time|Restarting the docker service may improve`,
 	// don't care if we can't push images to other profiles
 	`cache_images.go:.*error getting status`,
+	// network flakiness on VirtualBox
```

Comment:
I commend add example errors that you are whte listing so we know what we allow to the User to be spammed exactly.